plugin/src/main/java/com/yelp/codegen/KotlinGenerator.kt

```diff
@@ -131,13 +133,16 @@ class KotlinGenerator : SharedCodegen() {
     /** No testing files are needed on Kotlin Generator */
     override val testingSupportFiles = listOf<SupportingFile>()
 
-    override fun listTypeWrapper(listType: String, innerType: String) =
+    @VisibleForTesting
+    public override fun listTypeWrapper(listType: String, innerType: String) =
```

Comment:
kotlin default visibility is `public`, this change is making them private (according to the [doc](https://developer.android.com/reference/android/support/annotation/VisibleForTesting)).
Is this change an intentional change?
If so, then this change _breaks_ `KotlinGenerator` backward compatibility.

Options here are:
 * do not make the method private if not used in testing (~= revert this diff)
 * have a major version bump
 * have a minor bump highlighting the fact that there is a breaking change. NOTE: this option is actually going a bit beyond what sem-versioning defines but I do assume that the method was never assumed to be public

Reply:
> kotlin default visibility is public, this change is making them private (according to the doc).

Yup, default visibility is public, but not in this case.
Here the visibility is inherited from the super class (is `protected`) so I'm relaxing it to `public`. 

Given that the visibility is wider I don't think this is a backward incompatible change.